test/instrumentation/testdata/pkg/kubelet/metrics/metrics.go

```diff
@@ -269,6 +264,32 @@ var (
 		[]string{"resource_name"},
 	)
 
+	// TestSummary is a Summary that tracks the cumulative number of device plugin registrations.
+	// Broken down by resource name.
+	TestSummary = metrics.NewSummary(
+		&metrics.SummaryOpts{
+			Subsystem:      KubeletSubsystem,
+			Name:           "summary_metric_test",
+			Help:           "Cumulative number of device plugin registrations. Broken down by resource name.",
+			StabilityLevel: metrics.STABLE,
+		},
+	)
+	// TestSummaryVec is a NewSummaryVec that tracks the duration (in seconds) to serve a device plugin allocation request.
+	// Broken down by resource name.
+	TestSummaryVec = metrics.NewSummaryVec(
+		&metrics.SummaryOpts{
+			Subsystem:      KubeletSubsystem,
+			Name:           "summary_vec_metric_test",
+			Help:           "Duration in seconds to serve a device plugin Allocation request. Broken down by resource name.",
+			Objectives:     defObjectives,
+			MaxAge:         metrics.DefMaxAge,
+			AgeBuckets:     metrics.DefAgeBuckets,
+			BufCap:         metrics.DefBufCap,
+			StabilityLevel: metrics.STABLE,
+		},
+		[]string{"resource_name"},
+	)
```

Comment:
I feel like this isn't the right place to add new test metrics since this file is an old copy of https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/metrics/metrics.go and I would expect it not to diverge much from the original one

Reply:
You are missing the point. That file is used because it has a wide variety of metrics which makes it a good candidate for parsing. The current formulation is half-baked though since setting a stable metric in that file as it currently exists would actually create a stable metric.